pkgs/by-name/gm/gmsh/package.nix

```diff
@@ -2,6 +2,7 @@
   lib,
   stdenv,
   fetchurl,
+  fetchpatch,
```

Comment:
Probably not needed as you inlined the patch.

Reply:
Thx was left over from an earlier iteration when I'd originally written it with `fetchpatch` and then decided to add the patch file instead. Reverted back to the earlier `fetchpatch` version now.